src/main/java/Controllers/MainController.java

```diff
@@ -2,37 +2,43 @@
 
 import Controllers.Vista.VistaNavigator;
 import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXTextField;
+import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.Node;
-import javafx.event.ActionEvent;
 import javafx.scene.layout.BorderPane;
 import javafx.scene.layout.StackPane;
 import view.Controller;
 
-
-/**
- * Main controller class for the entire layout.
- */
 public class MainController {
 
-    protected static final String serverURL = "http://localhost:8080";
+    public static final String serverURL = "http://localhost:8080";
```

Comment:
Is this should be public? 
Since all the controllers should be in the same package.